### ISSUE TITLE
if shortcut is set for copy features then Ctrl+C not working anymore

### ISSUE THREAD

**Issue by MMR100** [qgis/QGIS#37401]
In QGIS 3.14, the Ctrl+C shortcutis not working anymore. 
As far as I understand, it has been removed in this version due to issues with Python Console. 
However I am not using the console so I decided to set up again Ctrl+C as a shortcut for copying features in keyboard shortcuts. 
The shortcut is not working. 
If I set up another shortcut (e.g. Ctrl+W), it is well working. 
I don't mind to use a new shortcut instead of the classic Crtl+C, but I think the user should be informed that Ctrl+C cannot be chose as a shortcut for copying features.

**Comment by nirvn** [comment]
@MMR100 , Ctrl+C is still working, but it's scope has been limited to avoid interference with panels. If you left click on the map canvas (with selected features) and click on Ctrl+C, features should be copied. Can you confirm?

**Comment by MMR100** [comment]
I found the error. 
If no shortcut is set for copy feature, the Ctrl+C is working. 
If I set up Ctrl+C in the list of keyboard shortcut, then it is not working anymore. 
Strange??

**Comment by MMR100** [comment]
I checked for Ctrl + X, Ctrl+V and Ctrl+A. The same problem occurs.

**Comment by troopa81** [comment]
I think it's a duplicate of #37503. I close the ticket, please re-open it if the problem still persists on master after #38810 merging

**Comment by thbaumann** [comment]
The problem still exists in QGIS 3.28 / 3.30 / 3.32. As soon as in the shortcut options of QGIS CTRL+C and CTRL+V is set then features won't be copied. As soon as you remove the shortcut-assignment it works again.

**Comment by MieWinstrup** [comment]
It is still a problem in QGIS 3.40 / 3.44 and 3.99 (the nightly build). I have tried with a clean user profile, and I'm only able to copy features from one layer to another layer with the shortcuts CTRL+C and CTRL+V - as long as I haven't set up a shortcut. That seems weird.

**Comment by troopa81** [comment]
@MieWinstrup I just tried it with current master and it works. Could you describe exactly what you are doing and share a project sample so I could reproduce ?

Let's reopen the issue in the mean time

**Comment by MieWinstrup** [comment]
Before I set up shortcuts, I can copy points from one layer and copy to another pointlayer with CTRL+C and CTRL+V.

When I set up a shortcut for Copy features and Paste features - then I can not use the shortcuts.

I have made a film showing the problem (I can't upload the mp4 directly to this issue): 

https://septima-kurser.s3.eu-central-1.amazonaws.com/Temp_filer/qgisproblem.mp4

**Comment by troopa81** [comment]
@MieWinstrup Thank you, I managed to reproduce the issue. I understand why it breaks.

May I ask why you need to set the shortcut to Ctrl+C/Ctrl+V while it's already the defined shortcut ?

Is it because the shortcut is not displayed in the menu? Or is there more?
